spec/formtastic_integration_spec.rb

```diff
@@ -0,0 +1,49 @@
+# rubocop:disable AlignArray
+require 'spec_helper'
+require 'formtastic'
+require 'formtastic/inputs/trix_editor_input'
+
+describe Trix do
+  include ActionPack
```

Comment:
If you are referring to the `include` lines, I'm sure we could check how rails does this part, and copy it, but I think that would include a lot more dependency which could slow down the tests, in the meantime I did a little clean up, by moving the formtastic related code into its own module, see the comment below.